packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -74,15 +74,13 @@ export class Cluster extends cdk.Construct implements ICluster {
    *
    * Returns the AutoScalingGroup so you can add autoscaling settings to it.
    */
-  public addDefaultAutoScalingGroupCapacity(options: AddDefaultAutoScalingGroupOptions): autoscaling.AutoScalingGroup {
-    const autoScalingGroup = new autoscaling.AutoScalingGroup(this, 'DefaultAutoScalingGroup', {
```

Comment:
"Default" is a wrong word maybe, also looking for better ideas here. 

The gist of it is that you don't have to:

* Configure VPC (automatically correct)
* Configure AMI (automatically correct)
* Other settings are set automatically for you as well (in case of EKS, `MaxPods` is derived from the instance type).

You might still want multiple sets of these, maybe but some in public and some in private subnets, maybe add two different fleets with two difference instance types.  It would be bad if you had to drop from the "we make it super pleasing" to "you have to do everything yourself" API if you wanted to add a 2nd autoscalinggroup.

Reply:
How about just `addCapacity`?